kuka_kr16_support/urdf/kr16r1610cybertech_macro.xacro

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="kuka_kr16r1610cybertech" params="prefix">
+    <!-- LINKS -->
+    <!-- base link -->
+    <link name="${prefix}base_link">
+      <inertial>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <mass value="2"/>
+        <inertia ixx="0.01" ixy="0" ixz="0" iyy="0.01" iyz="0" izz="0.01" />
+      </inertial>
```

Comment:
I don't believe we add these 'default' kind of `inertial` tags.

I understand you'd need this for Gazebo compatibility, but all values are fake/incorrect, and just by having these it could give the impression this `xacro:macro` can be successfully used with Gazebo.

Unless you feel strongly about keeping them, I'd suggest removing them.